src/modules/checkIsValidUserInput.js

```diff
@@ -9,7 +9,31 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  // Перевіряємо, чи введене значення є числом і має рівно 4 цифри
+  if (isNaN(userInput) || userInput.length !== 4) {
```

Comment:
The use of `isNaN(userInput)` may not work as intended because `isNaN` converts the input to a number before checking. This means non-numeric strings will be converted to `NaN`, resulting in `isNaN` returning `true`. Consider using a regular expression to ensure the input is a numeric string with exactly 4 digits.